x-pack/plugins/fleet/public/applications/fleet/components/search_bar.tsx

```diff
@@ -38,29 +34,42 @@ const NoWrapQueryStringInput = styled(QueryStringInput)`
 
 interface Props {
   value: string;
-  fieldPrefix?: string;
+  indexPattern: string;
+  fieldPrefix: string;
   onChange: (newValue: string, submit?: boolean) => void;
   placeholder?: string;
-  indexPattern?: string;
   dataTestSubj?: string;
 }
 
-const getMappings = (indexPattern: string) => {
+const getMappings = (indexPattern: string, fieldPrefix: string) => {
   switch (indexPattern) {
-    case `.${AGENTS_PREFIX}`:
+    case AGENTS_INDEX:
       return AGENT_MAPPINGS;
-    case `.${AGENT_POLICY_SAVED_OBJECT_TYPE}`:
-      return AGENT_POLICY_MAPPINGS;
-    case `.${PACKAGE_POLICY_SAVED_OBJECT_TYPE}`:
-      return PACKAGE_POLICIES_MAPPINGS;
-    case `.${FLEET_ENROLLMENT_API_PREFIX}`:
+    // Saved Objects are stored in .kibana_ingest.
+    // Currently, the search bar is only used to query agent policies.
+    case INDEX_NAME:
```

Comment:
nit: `INDEX_NAME` seems too generic, we could use `INGEST_SAVED_OBJECT_INDEX` which refers to the same constant.
https://github.com/elastic/kibana/blob/3ae7ba5ad9c82c51d38448fe07d6dba6c6b2627c/x-pack/plugins/fleet/public/constants/index.ts#L35